modules/couchbase/src/main/java/org/testcontainers/couchbase/UserDefinition.java

```diff
@@ -0,0 +1,56 @@
+package org.testcontainers.couchbase;
```

Comment:
I think this needs a license header

Reply:
I assume that because this is a new addition and my contribution I'm free to use the MIT license?

Reply:
yes, let's keep this class custom license free. We did allow a custom license when the initial module was contributed, but we prefer to stick to the "main" license we have.

Reply:
The other files used apache - I'm not sure about the license, but I bet @rnorth and @bsideup can help us here

Reply:
I'm going to trust stackexchange on this one :)

https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project

Reply:
ah ok you guys beat me to it

Reply:
I noticed that there is no LICENSE.txt/md in the root of the couchbase module, I wonder if that should be added to reflect that all/most of the couchbase code is apache licensed vs MIT which is in the root of the testcontainers-java repo..

Reply:
@daschl we do not use license headers, let's keep it consistent